packages/components/src/Interactive.css

```diff
@@ -0,0 +1,19 @@
+.hover-style:hover, .hover-style:focus {
```

Comment:
to guard against css class name collisions, lets prefix these style names with something like terra-ui--hover-style

Reply:
Renamed inhttps://github.com/DataBiosphere/terra-ui/pull/4147/commits/b9b700c80794b89c76db6b093240aaeeee249ec9.